src/java/com/unifina/feed/FieldDetector.java

```diff
@@ -37,33 +58,59 @@ public void setFlattenMap(boolean flattenMap) {
 			map = new HashMap();
```

Comment:
Variable map is defined above without the types it holds:
Should read: Map<String, Object> map;